### ISSUE TITLE
When using `safeBothPrefix` the prefix `[dir]` is missing for styles using `padding-block`

### ISSUE THREAD

**Issue by edba** [elchininet/postcss-rtlcss#222]
When using `padding-block` and `safeBothPrefix` the output doesn't wrap the class using `padding-block` with `[dir]` that leads to specificity issues with other styles using `padding`.

Example
```
.test {
    padding-block: 10;
}

input {
    padding: 0;
}
```

Output:
```
.test {
    padding-block: 10;
}

[dir] input {
    padding: 0;
}
```

Expected:
```
[dir] .test {
    padding-block: 10;
}

[dir] input {
    padding: 0;
}
```

Playground: https://elchininet.github.io/postcss-rtlcss/#6503a1cc884f3

**Comment by elchininet** [comment]
Hi @edba,
I'll take a look when I get some time.
Regards

**Comment by elchininet** [comment]
Hi @edba,
The issue has been fixed in [the version 4.0.8](https://www.npmjs.com/package/postcss-rtlcss/v/4.0.8). You can test it with [the same link](https://elchininet.github.io/postcss-rtlcss/#6503a1cc884f3) that you provided before.
Regards
